### ISSUE TITLE
Difference between dataset_tag filter and ratio-0.01/ratio-0.1

### ISSUE THREAD

**Issue by yudhiesh** [myscale/benchmark#14]
I am running the benchmarks myself and I have noticed that I don't have the option to filter down by ratio. Isn't the ratio here a filter as well which shows the ratio of results returned when performing a filter by specifc query files?

I ran my benchmark with the argument `--datasets laion-768-5m-ip-probability`. 

*What I see*
![Screenshot 2024-05-23 at 11 26 00 AM](https://github.com/myscale/benchmark/assets/55042754/7d2b1d32-2053-4625-bec6-1fd84f0ad3e4)

*What I see in the website*
![Screenshot 2024-05-23 at 11 27 52 AM](https://github.com/myscale/benchmark/assets/55042754/e9b408d2-edad-4ebb-b47d-6b2dd7f72e3b)

I have downloaded all the datasets as well:
![Screenshot 2024-05-23 at 11 29 28 AM](https://github.com/myscale/benchmark/assets/55042754/831ad1bf-5b14-4fef-a7b2-7f27043d2704)

**Comment by MochiXu** [comment]
Sorry, I missed the script to handle the filter dataset. You need to pull the vector-db-benchmark on the master branch again. See the [link](https://github.com/myscale/vector-db-benchmark/tree/master/scripts) for details on using the script

**Comment by yudhiesh** [comment]
> Sorry, I missed the script to handle the filter dataset. You need to pull the vector-db-benchmark on the master branch again. See the [link](https://github.com/myscale/vector-db-benchmark/tree/master/scripts) for details on using the script

So I just have to run the `handle_probability.py` on the `results` that I already have generated?

**Comment by MochiXu** [comment]
@yudhiesh  That's right, the script `handle_probability.py` generates different folders based on each filter ratio. These folders might be `results-0.001`, `results-0.01`, and `results-0.1`. The generated folder with the filter ratio suffix is in the same folder as the results directory, and then you only need to use the `process_result_files.py` file to process the folder with the filter ratio suffix.

**Comment by yudhiesh** [comment]
I tried running it and get the following result directories:
```
results/
results-0.1/
results-0.0.1/
```
But the benchmark results are only showing the filter results. I did update the path [here](https://github.com/myscale/vector-db-benchmark/blob/f342baf387db036e1148438d3e3da6d600984b62/scripts/process_result_files.py#L78) to include `results` as well. 

EDIT:
After manually removing all the `probabiliy` results in the original `results` directory and running the `process_results_file.py` it shows up now.
